src/gwt/src/org/rstudio/studio/client/workbench/prefs/model/UserPrefsAccessor.java

```diff
@@ -401,6 +401,9 @@ public final native boolean getConsoleRightOnTop() /*-{
       }-*/;
 
       public final native int getAdditionalSourceColumns() /*-{
+         if (!this.additional_source_columns)
+           return 0;
+
```

Comment:
@melissa-barca This was returning undefined for me and I couldn't open the global options. Unfortunately Java doesn't have the tools to check if a primitive type is undefined. Transpiling is so 🤦. I noticed that you were checking if this was defined in other places but I think because of the order of Ginjection this was undefined for me higher up the chain. Not sure if you would prefer to tackle this in a different way.

Reply:
Looks good thank you for updating this 👍